Dockerfile.daemon.openshift

```diff
@@ -1,4 +1,8 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+RUN yum install -y libbpf libbpf-devel libxdp-1.2.1-1.el8.x86_64
+# Temporary workaround until https://issues.redhat.com/browse/ENGCMP-2609 is fixed.
+RUN yum install -y \
+  https://koji.mbox.centos.org/pkgs/packages/xdp-tools/1.2.1/1.el8/x86_64/libxdp-devel-1.2.1-1.el8.x86_64.rpm
```

Comment:
is this a legit place for this pkg , don't we have it brew ?

Reply:
No it's not a valid location. We can't pull it from brew because brew is behind the VPN and the build system doesn't have access to it as mentioned yesterday during the meeting [0]. I think I will follow Toke's approach to just pull in the *.h files in some location and use that?

[0] everything that I had tried yesterday:
~~~
# 0) Doesn't work because xdp/libxdp.h missing (contained in libxdp-devel):
# RUN yum install -y libbpf libbpf-devel libxdp xdp-tools
# a) Doesn't work due to https://bugzilla.redhat.com/show_bug.cgi?id=2129035 and also because we need 1.2.6.1 which will
# only ship with RHEL 8.7:
# RUN yum install -y libbpf libbpf-devel libxdp libxdp-devel
# b) Directly pull XDP RPMs from brew until https://bugzilla.redhat.com/show_bug.cgi?id=2129035 is fixed.
# Only works on my local system within the VPN, not with the build system :-(
# RUN yum install -y libbpf libbpf-devel
# RUN yum install -y \
#  http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-1.2.6-1.el8.x86_64.rpm \
#  http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-devel-1.2.6-1.el8.x86_64.rpm
# c) Try building from SRPMs: Doesn't work as no SRPMs enabled in the builder image:
# RUN yum install -y libbpf libbpf-devel
# RUN yum download --source xdp-tools-1.2.1-1.el8.src
# RUN rpmbuilder -b xdp-tools*.rpm
# d) Use the CentOS RPMs ...
# RUN yum install -y libbpf libbpf-devel
# RUN yum install -y \
#  https://koji.mbox.centos.org/pkgs/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-1.2.6-1.el8.x86_64.rpm \
#  https://koji.mbox.centos.org/pkgs/packages/xdp-tools/1.2.6/1.el8/x86_64/libxdp-devel-1.2.6-1.el8.x86_64.rpm
# e) Use a mix ...
RUN yum install -y libbpf libbpf-devel libxdp-1.2.1-1.el8.x86_64
# Temporary workaround until https://issues.redhat.com/browse/ENGCMP-2609 is fixed.
RUN yum install -y \
  https://koji.mbox.centos.org/pkgs/packages/xdp-tools/1.2.1/1.el8/x86_64/libxdp-devel-1.2.1-1.el8.x86_64.rpm
~~~

Reply:
try this 
```
RUN dnf install -y libbpf \
  http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/xdp-tools/1.2.1/1.el8/x86_64/libxdp-1.2.1-1.el8.x86_64.rpm
```